### ISSUE TITLE
Naming inconsistency for Amino Acid HGVS expressions

### ISSUE THREAD

**Issue by malachig** [BRL-BCM/Allele-Registry#5]
Thank you for the ClinGen Allele Registry.  An absolutely invaluable resource!

One thing that has caught my attention that would be nice to clarify/fix.  Consider this example:
https://reg.genome.network/redmine/projects/registry/genboree_registry/by_caid?caid=CA180983

One transcript HGVS for this allele looks like this.  
`ENST00000275493.7:c.2311_2319dup (EGFR)` 

To the right we see the corresponding protein representation shown as:
`ENSP00000275493.2:p.His773_Val774insAsnPr...` 

As you can see it is truncated for display purposes due to length.  But you can follow the URL to a dedicated page for this `p.`

However, on that page it is shown as:
https://reg.genome.network/redmine/projects/registry/genboree_registry/by_caid?caid=PA2573051107
`ENSP00000275493.2:p.Asn771_His773dup`

In other words you never get to see the full length version of `ENSP00000275493.2:p.His773_Val774insAsnPr...`.  It would be nice to see this. Or one might decide to show the `ENSP00000275493.2:p.Asn771_His773dup` in both places as this is the desired convention according to HGVS anyway?
